recipes/codecov/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "codecov" %}
+{% set version = "2.0.6" %}
+{% set sha256 = "999bf4d6f7e072433601798f3f291277a4ab519b245af2971cdeb247dbb21454" %}
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  # Not on pypi (at time of packaging)
+  url: https://github.com/codecov/codecov-python/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - setup.py.patch
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - setuptools
+    - coverage
+    - requests >=2.7.9
+
+test:
+  imports:
+    - codecov
+  commands:
+    - codecov -h
+
+about:
+  home: https://github.com/codecov/codecov-python
+  license: Apache-2.0
+  license_family: Apache
```

Comment:
Please ask upstream to include a license file. Also please ask that it be included in `MANIFEST.in`.